f/assignGear/f_assignGear_3IFB.sqf

```diff
@@ -374,7 +374,7 @@ if (_isMan) then {
 
 // Standard Loadout
 // if (_loadout == 1) then {
-	#include "f_assignGear_3IFB_standard.sqf"
+	#include "f_assignGear_3IFB_standard.sqf";
```

Comment:
Is there a reason why we sometimes(?) use semicolons here? I don't think they're needed?

Reply:
We always (as far as I can see) use them after #includes that are "in the open". They were only missing here because these lines were converted from if cases that don't use them.